connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java

```diff
@@ -48,7 +50,11 @@ public ConnectorPluginsResource(Herder herder) {
     @Path("/{connectorType}/config/validate")
```

Comment:
Should we deprecate `/{connectorType}/config/validate` in favor of `/config/validate`?

Alternately, maybe we should behave similarly to the `putConnectorConfig` resource where if the name is included in the map, we verify that it's the same as in the path, otherwise we insert it into the map.